### ISSUE TITLE
hardware question for the compliant gripper

### ISSUE THREAD

**Issue by Blade-B3AR** [IVProjects/Engineering_Projects#15]
What hardware is needed for assembly of the compliant gripper

**Comment by IVProjects** [comment]
Hi, I assume you're the same person that commented about being a NASA HUNCH student. The 3D printable files are located within the servoArmV2 folder, but I assume you already found those. The electronics are an SG90 servo, a 5mm LED, a photoresistor, and a simple breadboard pushbutton for the tension sensor.

Could you tell me a little more about the project you're using this for? I know HUNCH sometimes sends things up to the ISS. Is that what this project is for?

If it would help I can also provide the solidworks model for the gripper and the code.

**Comment by Blade-B3AR** [comment]
yes I am just g

**Comment by Blade-B3AR** [comment]
yes i am, we are working on the mobility arm for a robotic duster for the iss [info here](https://www.hunchdesign.com/uploads/2/2/0/9/22093000/robotic_duster_--mobility_arm_10-4.pdf) but yes i did find the files just wasn't sure what all screws and such that were used but we were thinking of using your gripper for holding onto the handrails in the iss as we seen you're using string for closing it which is very similar to how we planned on moving the arm so we thought it would be a perfect fit for it [pictures if your curious](https://photos.app.goo.gl/5R92uhLoJZEX8Q1PA) if you would be willing to provide them that would be great just in case we have to change anything, although we are using inventor and fusion but i can proably get them to work still

**Comment by IVProjects** [comment]
In terms of screws -- the screws holding the servo in place are the ones supplied with the servo. The fingers are just pressed in place so no fasteners are needed. The string is sewing thread. I printed all the parts with PLA filament, but if you have access to PETG filament I think that would work better for the fingers. 

Thanks for the info about the project. I'm willing to help out if you run into any issues. I'll add the solidworks models and code when I have a chance. Good luck!

**Comment by Blade-B3AR** [comment]
Okay thank you so much! If you want I can let you know if we advance to finals and such

**Comment by Blade-B3AR** [comment]
not to bother but any word on those model files

**Comment by IVProjects** [comment]
Sorry about that, I got busy with finals and forgot. I just added the solidworks models under the compliant gripper folder with the STLs. 

Please do let me know if you move onto finals!
